### ISSUE TITLE
Về Database

### ISSUE THREAD

**Issue by ndb1997vn** [BuiHuongGiang0410/Nhom21-CNPM#13]

**Comment by Chamiko** [comment]
Alo, bên server cho xin cái CSDL mà hôm nọ mới hoàn thành hơn 200 bản ghi ấy để convert sang .sqlite import thử vào project xem cái chạy offline nó ổn chưa

**Comment by anhbh310** [comment]
Gmail chung cho nhóm làm việc từ việc hosting, database, youtube,... mọi người dùng cái này nhé, myteamhus1997@gmail.com, pass mình đưa mọi người hết rồi

**Comment by anhbh310** [comment]
Còn CSDL thì mai Huy với Hiếu gửi lên cho

**Comment by Chamiko** [comment]
Ok, với cả mọi người lấy cái gmail đấy vào youtube, t có tạo 1 cái playlist https://www.youtube.com/playlist?list=PLIqf0aj-QeToj2hsz6cxnuSO4hqtB_GEx.
Tên Let's Cook ấy, mỗi người loanh quanh youtube xem video nấu ăn nào hay hay chất lượng thì add vào playlist giúp t với, để cho nó phong phú.

**Comment by ndb1997vn** [comment]
Yup. Cứ thêm tạm trước 1 vài cái để test đi, rồi rảnh rảnh ae tìm thêm cho :)

**Comment by PhanVietHuy** [comment]
Đây nhé, down về giải nén ra
https://goo.gl/u5Wwtf

**Comment by Chamiko** [comment]
okela

**Comment by PhanVietHuy** [comment]
![image](https://user-images.githubusercontent.com/32640151/34026894-b35c67b4-e18b-11e7-876b-f3e11dc219b0.png)
xóa cái cột blob đi đúng k?
